Authentication.md

```diff
@@ -619,6 +620,100 @@ if err != nil {
 ```
 
 
+## Multi-Cloud Saas Platform (MCSP) Authentication
+The `MCSPAuthenticator` can be used in scenarios where an application needs to
+interact with an IBM Cloud service that has been deployed to a non-IBM Cloud environment (e.g. AWS).
+It accepts a user-supplied apikey and performs the necessary interactions with the
+Multi-Cloud Saas Platform token service to obtain a suitable MCSP access token (a bearer token)
+for the specified apikey.
+The authenticator will also obtain a new bearer token when the current token expires.
+The bearer token is then added to each outbound request in the `Authorization` header in the
+form:
+```
+   Authorization: Bearer <bearer-token>
+```
+
+### Properties
+
+- ApiKey: (required) the apikey to be used to obtain an MCSP access token.
+
+- URL: (required) The URL representing the MCSP token service endpoint's base URL string. Do not include the
+operation path (e.g. `/siusermgr/api/1.0/apikeys/token`) as part of this property's value.
+
+- DisableSSLVerification: (optional) A flag that indicates whether verificaton of the server's SSL 
+certificate should be disabled or not. The default value is `false`.
+
+- Headers: (optional) A set of key/value pairs that will be sent as HTTP headers in requests
+made to the MCSP token service.
+
+- Client: (optional) The `http.Client` object used to invoke token service requests. If not specified
+by the user, a suitable default Client will be constructed.
+
+### Usage Notes
+- When constructing an MCSPAuthenticator instance, you must specify the ApiKey and URL properties.
+
+- The authenticator will use the token server's `POST /siusermgr/api/1.0/apikeys/token` operation to
```

Comment:
The readme looks good to me, although I am curious as to what this `/siusermgr/api/1.0/apikeys/token` endpoint is. Is this something users will need to be worried about or could potentially want to change?

Reply:
No, that's just the "path" associated with the operation that the authenticator invokes.  I just added that statement so that curious users (and developers) would know which specific operation the authenticator is using to do the token exchange.